src/app.rs

```diff
@@ -758,3 +763,8 @@ pub fn build_app() -> App<'static, 'static> {
 
     app
 }
+
+#[test]
+fn verify_app() {
+    build_app().debug_assert()
```

Comment:
what does this do?

Reply:
It tests that all the debug asserts in building the `App` succeed. Adding this test was recommded in the migration guide.